docs/superpowers/plans/20260817_1030_保存容量の上限とエビクション.md

```
@@ -0,0 +1,2204 @@
+# 保存容量の上限とエビクション（PR 3）実装計画
+
+> **For agentic workers:** REQUIRED SUB-SKILL: Use superpowers:subagent-driven-development (recommended) or superpowers:executing-plans to implement this plan task-by-task. Steps use checkbox (`- [ ]`) syntax for tracking.
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)